mllib/src/main/scala/org/apache/spark/ml/tuning/ParamGridBuilder.scala

```diff
@@ -100,10 +100,11 @@ class ParamGridBuilder {
    * Builds and returns all combinations of parameters specified by the param grid.
    */
   def build(): Array[ParamMap] = {
-    var paramMaps = Array(new ParamMap)
+    var paramMaps = Array.empty[ParamMap]
```

Comment:
Yes, though I'd put braces in both branches and indent if writing it this way, or just handle it all in an early return statement.